panels/user-accounts/um-app-permissions.c

```diff
@@ -54,7 +55,7 @@ struct _UmAppPermissions
 
   GCancellable *cancellable; /* (owned) */
   EpcAppFilter *filter; /* (owned) */
-  guint         selected_age;
+  guint         selected_age;  /* `(guint32) -1` to disable OARS */
```

Comment:
Nitpick: remove one space before `/*`

Reply:
I was using the Python style, but whatever.